doc/test-framework/scorecard.md

```diff
@@ -191,147 +194,230 @@ to the console if the scorecard is being run in with `output` unset or set to `t
 
 ### JSON format
 
-The JSON output is formatted in the same way that a Kubernetes API would be, which allows for updates to the schema as well as the use of various Kubernetes helpers. The Golang structs are defined in `pkg/apis/scorecard/v1alpha1/types.go` and can be easily implemented by plugins written in Golang. Below is the JSON Schema:
+The JSON output is formatted in the same way that a Kubernetes API would be, which allows for updates to the schema as well as the use of various Kubernetes helpers. The Golang structs are defined in `pkg/apis/scorecard/v1alpha2/types.go` and can be easily implemented by plugins written in Golang. Below is the JSON Schema:
 
 ```json
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
   "$ref": "#/definitions/ScorecardOutput",
   "definitions": {
-    "ScorecardOutput": {
```

Comment:
Is not it a break change?

Reply:
yep, I removed the v1alpha1 since v1alpha2 is now the default and v1alpha1 is going to be removed/deprecated at some point.

Reply:
In this scenario, I think the best would keep the v1alpha1 with the info **deprecated** and add the  `v1alpha2`. I think we can just remove the docs over v1alpha1 after we finish v1alpha2 but I am not sure so. @joelanford, could you gve a hand here, how we should proceed about it?

Reply:
I think we should completely remove `v1alpha1` as soon as possible (which is when CVP has moved to v1alpha2). In the meantime, I think it is okay to remove these docs about v1alpha1.

Since we're still in alpha versions, the expectation is that the APIs can and will change between versions, so I don't think there's a huge necessity to keep `v1alpha1` around as deprecated, especially since it is a hindrance to our scorecard refactoring efforts.